djangocms_picture/cms_plugins.py

```diff
@@ -68,8 +68,8 @@ def render(self, context, instance, placeholder):
         # assign link to a context variable to be performant
         context['picture_link'] = instance.get_link()
         context['picture_size'] = instance.get_size(
-            width=context.get('width'),
-            height=context.get('height'),
+            width=float(context.get('width')),
```

Comment:
no width in context returns None which will blow up when passed to float().
Same with height.